### PR TITLE
fix/lint_error_tests, fix eslint wornings in tests

### PR DESCRIPTION
**Pull request by AndrZakhara**
Eslint errors fixed. 
I'm waiting for your review.
Thanks

### CODE REVIEW

File: src/tests/reduce.test.js [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
   });
 
-  test("Method shouldn't mutate initial array", () => {
+  test('Method shouldn\'t mutate initial array', () => {
     const arr = new MyArray(1, 2, 3);
     const mockCallback = jest.fn();
     arr.reduce(mockCallback);
-
-    expect(arr).toEqual(MyArray(1, 2, 3));
+    const newArray = new MyArray(1, 2, 3);
```
Comment (Jarosic) [inline src/tests/reduce.test.js]: Why did you put this expression into a separate variable

File: src/tests/instance.test.js [hunk 2]
```diff
@@ -24,7 +24,9 @@ describe('tests for instance', () => {
   });
 
   test('Prototype have only declarated method and constructor', () => {
-    const declaratedMethods = ['constructor', 'find', 'slice', 'push', 'pop', 'toString', 'map', 'filter', 'forEach', 'reduce', 'sort', Symbol(Symbol.iterator)];
+    const declaratedMethods = ['constructor', 'find', 'slice', 'push', 'pop',
```
Comment (Jarosic) [inline src/tests/instance.test.js]: Write each element of the array with a new line

**Review by Jarosic — CHANGES_REQUESTED** [review]
Read my comment and make changes

File: src/tests/instance.test.js (hunk 2 above)
Comment (AndrZakhara) [reply src/tests/instance.test.js]: fixed

File: src/tests/reduce.test.js (hunk 1 above)
Comment (AndrZakhara) [reply src/tests/reduce.test.js]: fixed

**Review by Jarosic — APPROVED** [review]
Well done

**Review by Arkant — APPROVED** [review]
Nice

**Pull request merged**